browser/src/Config.ts

```diff
@@ -107,3 +126,22 @@ export function getValue<T>(configValue: string): T {
 export function getUserFolder(): string {
     return path.join(Platform.getUserHome(), ".oni")
 }
+
+let listeners: Function[] = []
```

Comment:
For this, we don't actually need to create our own registration mechanism - we could just hijack the event emitter:

```
import { EventEmitter } from "events";
...
const configEvent = new EventEmitter()

export function registerListener(callback: Function): void {
configEvent.on("config-update", callback);
}

function notifyListeners(): void {
configEvent.emit("config-update")
}
```